bentoml/server/bento_api_server.py

```diff
@@ -289,4 +290,6 @@ def start(self):
         """
         Start an REST server at the specific port on the instance or parameter.
         """
+        track_server('flask')
```

Comment:
this event will be duplicated when running gunicorn server right?

Reply:
anyway to workaround?

Reply:
It is duplicated in each workers for gunicorn. I thought about putting it at server initialization. However I really want uptime for it, so it ends up on each worker. 

It’s is bad send the message multiple times. While I think it is kinda neat data point for how many workers

Reply:
Open for suggestions though